screeps-game-api/src/constants.rs

```diff
@@ -199,6 +209,13 @@ pub mod find {
         MINERALS, 116, Mineral;
         NUKES, 117, Nuke;
         TOMBSTONES, 118, Tombstone;
+        POWER_CREEPS, 119, PowerCreep;
+        MY_POWER_CREEPS, 120, PowerCreep;
+        HOSTILE_POWER_CREEPS, 121, PowerCreep;
+        EXIT_TOP, Exit::Top as i32, RoomPosition;
+        EXIT_RIGHT, Exit::Right as i32, RoomPosition;
+        EXIT_BOTTOM, Exit::Bottom as i32, RoomPosition;
+        EXIT, Exit::All as i32, RoomPosition;
```

Comment:
These should help with the situation we last talked about in the slack chat!

I might be putting too much in this PR. But what else is an "overhaul" for?